dv/uvm/yaml/rtl_simulation.yaml

```diff
@@ -88,3 +88,22 @@
       -covbaserun test
     wave_opts: >
       -input <cwd>/ius.tcl
+
+- tool: riviera
+  env_var: ALDEC_PATH
+  compile:
+    cmd:
+      - "vlib <out>/work"
+      - "vlog -work <out>/work
+        +incdir+<ALDEC_PATH>/vlib/uvm-1.2/src
+        +incdir+${PRJ_DIR}/ibex/shared/rtl
+        -l uvm_1_2
+        -err VCP2694 W1
```

Comment:
This argument is needed to lower severity for compiler error on illegal assignment to enum to warning.
All line where illegal assignment are done is now marked with warning ie:
WARNING VCP2694 "Assignment to enum variable from expression of different type." "/home/danielm/gitlab/ibex/dv/uvm/tests/core_ibex_test_lib.sv" 781  1

This argument will be no longer needed when explicit cast will be used on such illegal assignment ie in file :
 /home/danielm/gitlab/ibex/dv/uvm/tests/core_ibex_test_lib.sv, line 201
` operating_mode = core_init_mstatus[12:11];`
need to be changed to:
` operating_mode = priv_lvl_e'(core_init_mstatus[12:11]);`

Reply:
It seems like our lint flow excludes DV code, I've filed https://github.com/lowRISC/ibex/issues/569 to get that sorted. Thanks for letting us know!